src/mail/editor/MailEditor.ts

```diff
@@ -771,36 +771,36 @@ async function createMailEditorDialog(model: SendMailModel, blockExternalContent
 	}
 
 	const minimize = () => {
-		// If the mail is unchanged, close instead of saving
-		if (!model.hasMailChanged()) {
+		if (model.hasMailChanged() || model.draft) {
```

Comment:
we probably don't want to save if there are no changes though, just showing the minimized editor is enough